### PR TITLE
Player page enhancements

### PR DESCRIPTION
**Pull request by ceoaliongroo**
Finish 4 of 7 task #58

### CODE REVIEW

File: app/scripts/controllers/playMarker.js [hunk 1]
```diff
@@ -16,8 +17,19 @@ angular.module('publicEducationApp')
     $scope.playList = [];
     $scope.currentTrack = 0;
 
+    // Default values, user.
+    $scope.user = {
+      name:"Anonymous",
+      email: ''
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: Remove.

File: app/scripts/controllers/playMarker.js [hunk 2]
```diff
@@ -74,4 +89,41 @@ angular.module('publicEducationApp')
...
+      var url;
+      var text = $scope.selectedMarker.currentRecord.text + '-' + encodeURIComponent($location.absUrl());
+
+      if (method === 'twitter') {
+        url = 'https://twitter.com/share?text='+text;
+      }
+      else if (method === 'facebook') {
+        url = 'http://www.facebook.com/sharer/sharer.php?s=100&p[url]='
+          + encodeURIComponent($location.absUrl())
+          + '&p[images][0]=&p[title]=Public%20Education&p[summary]=' + text;
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: Try removing `[images][0]`

File: app/scripts/controllers/playMarker.js [hunk 3]
```diff
@@ -74,4 +89,41 @@ angular.module('publicEducationApp')
...
+      }
+      else if (method === 'email') {
+        url = 'mailto:'
+          + $scope.user.email
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: always empty.

File: app/scripts/controllers/playMarker.js [hunk 4]
```diff
@@ -74,4 +89,41 @@ angular.module('publicEducationApp')
...
+          + '?body='
+          + text
+          + ' - '
+          + encodeURIComponent($location.absUrl());
+      }
+      else if (method === 'link') {
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: Remove in favor of a simple `< a href="" target="_blank">`

**Comment by ceoaliongroo** [comment]
@amitaibu Reviewed.

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**